pkg/controllers/management/clusteroperator/utils.go

```diff
@@ -251,3 +264,39 @@ func (e *OperatorController) RecordCAAndAPIEndpoint(cluster *mgmtv3.Cluster) (*m
 
 	return currentCluster, err
 }
+
+func GenerateSAToken(restConfig *rest.Config) (string, error) {
+	clientSet, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return "", fmt.Errorf("error creating clientset: %v", err)
+	}
+
+	return util.GenerateServiceAccountToken(clientSet)
+}
+
+func addAdditionalCA(secretsCache wranglerv1.SecretCache, caCert string) (string, error) {
+	additionalCA, err := getAdditionalCA(secretsCache)
+	if err != nil || additionalCA == nil {
```

Comment:
yet another very minor nit: this would be more readable split up. I know its more lines of code and another operation, but this looks like a plain error return, when really its error return _or_ a no-op.

Reply:
Done.